Pipfile

```diff
@@ -6,9 +6,14 @@ name = "pypi"
 [packages]
 datadog = "*"
 black = "*"
+jsonschema = "*"
+requests = "*"
 
 [dev-packages]
 flake8 = "*"
 
 [requires]
 python_version = "3.7"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Why do we need this?

Reply:
I hit https://github.com/pypa/pipenv/issues/1802 with ```black```.

Reply:
Specifically, see the output in https://github.com/mozilla/wpt-api/pull/236#pullrequestreview-202417891.